apps/src/componentLibrary/tabs/Tabs.tsx

```diff
@@ -0,0 +1,113 @@
+import classnames from 'classnames';
+import React, {useCallback, useState} from 'react';
+
+import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
+import _Tab, {TabModel} from '@cdo/apps/componentLibrary/tabs/_Tab';
+import TabPanel from '@cdo/apps/componentLibrary/tabs/TabPanel';
+
+import moduleStyles from './tabs.module.scss';
+
+export interface TabsProps {
+  /** Array of props for Tabs to render */
+  tabs: TabModel[];
+  /** The function that is called when a Tab is clicked/selected tab is changed */
+  onChange: (value: string) => void;
+  /** The value of the default selected Tab */
+  defaultSelectedTabValue: string;
+  /** The name attribute specifies the name of a Tabs group.
+     The name attribute is used to reference elements in a JavaScript.
+     */
```

Comment:
nit: comment formatting

Reply:
That's how our linter/prettier formats comment